src/Mapper/ORM/DoctrineORMMapper.php

```diff
@@ -66,13 +64,13 @@ public function getSubscribedEvents(): array
         ];
     }
 
-    public function addAssociation(string $class, string $field, OptionsBuilder $options): void
+    public function addAssociation(string $class, string $type, array $options): void
```

Comment:
Can you find a BC way for this?

Reply:
I think the only way is to remove the previous typing, but not adding the correcto one. I thought that since the bug is caused by this incorrect type we could change it without BC, because the previous type couldn't work on a real app. wdyt?